charts/incubator/kubernetes-reflector/values.yaml

```
@@ -0,0 +1,75 @@
+image:
+  # TODO: update image to use TC repo
+  repository: emberstack/kubernetes-reflector
+  pullPolicy: IfNotPresent
+  tag: 7.1.210
+
+operator:
+  register: true
+
+portal:
+  open:
+    enabled: false
+
+rbac:
+  main:
+    enabled: true
+    primary: true
+    clusterWide: true
+    rules:
+      - apiGroups:
+          - ""
+        resources:
+          - "configmaps"
+          - "secrets"
+        verbs:
+          - "get"
+          - "list"
+          - "watch"
+          - "create"
+          - "update"
+          - "patch"
+          - "delete"
+      - apiGroups:
+          - ""
+        resources:
+          - "namespaces"
+        verbs:
+          - "watch"
+          - "list"
+
+serviceAccount:
+  main:
+    enabled: true
+    primary: true
+
+
+kubernetesReflector:
+  logLevel: Information
+
+workload:
+  main:
+    podSpec:
+      automountServiceAccountToken: true
+      containers:
+        main:
+          securityContext:
+            readOnlyRootFilesystem: true
+            runAsNonRoot: true
+          env:
+            ES_Serilog__MinimumLevel__Default: Information
```

```suggestion
            ES_Serilog__MinimumLevel__Default: {{ .Values.kubernetesReflector.logLevel }}
```
You can do that directly! And remove from `common.yaml` the `set` command!